bazel/envoy_library.bzl

```diff
@@ -69,6 +69,14 @@ def envoy_cc_library(
         }),
         strip_include_prefix = strip_include_prefix,
     )
+    native.cc_library(
```

Comment:
Can you add a comment explaining what this is for and how it should be used (i.e. only by external projects)? Otherwise LGTM.

Reply:
Done!